test/unit/org/apache/cassandra/io/sstable/keycache/KeyCacheTest.java

```diff
@@ -396,7 +395,6 @@ public void testKeyCacheLoadTwoTablesTime() throws Exception
         // be zero after load.
         assertKeyCacheSize(numberOfRows, KEYSPACE1, columnFamily1);
         assertKeyCacheSize(numberOfRows, KEYSPACE2, columnFamily2);
-        assertEquals(sstableImplCachesKeys ? numberOfRows * tables.size() : 0, CacheService.instance.keyCache.size());
```

Comment:
why is this removed (above as well)? is that redundant? What is the relationship of this change to this ticket?